test/e2e/app-dir/index.test.ts

```diff
@@ -929,32 +915,6 @@ describe('app dir', () => {
           })
         })
 
-        describe('previewData function', () => {
```

Comment:
Are these test removals because the test suite in Next.js removed them?

Reply:
no, we need to overhaul our tests because our current scaffolding doesn't work with many of the new app-dir tests - i think it's important to get a basic level of support shipped and then work on getting these tests in, but shout if you disagree

Reply:
Sounds good. I created #2161 for this.